source/faq.txt

```diff
@@ -12,6 +12,27 @@ Frequently Asked Questions
    :depth: 2
    :class: singlecol
 
+How to fix MongoServerSelectionError: connect ECONNREFUSED ::1:27017 error?
```

Comment:
Issue: the noun requires an article ("a" or "the"). Also, could be easier to identify the error text.

Suggestion: separate the error text using quotation marks.
```suggestion
How to fix a "MongoServerSelectionError: connect ECONNREFUSED ::1:27017" error?
```